### DISCUSSION TITLE
Thoughts on swapping Enquirer for Inquirer?

### DISCUSSION THREAD

**Discussion by khill-fbmc** [hongaar/bandersnatch#409] [Ideas]
https://github.com/hongaar/bandersnatch/blob/144fc3e35cb0321379c26dbf742d8834b010ca09/src/prompter.ts#L1

I saw [here in your code that you are waiting for upstream changes](https://github.com/hongaar/bandersnatch/blob/main/src/prompter.ts#L15)

So I [visited the link you had in your source](https://github.com/enquirer/enquirer/pull/258) and it's been over a year. I don't think they will be updating it...

[I thought I was trying to show interest and see if they were active still](https://github.com/enquirer/enquirer/issues/401) but I was brushed off with "No"

Well, all the commits are 
![image](https://user-images.githubusercontent.com/107003416/180455021-25541139-dc90-4a25-9ec4-54edb0fc43fd.png)

It seems to me that [switching to Inquirer](https://github.com/SBoudrias/Inquirer.js) might be the better option? They at least seem to be actively maintaining their project.

Take it all with a grain of salt, these are just my musings.

**Comment by hongaar** [comment]
Thanks for the suggestion. It struck me as well that enquirer is indeed not very actively maintained. I switched from inquirer at the time to increase performance, see https://github.com/hongaar/bandersnatch/pull/81. You're welcome to revert these changes in a PR and benchmark the performance hit, maybe it has improved in the last years?
